### PR TITLE
Fix issues where fixed PTH files were written as UNICODE rather than UTF-8

### PR DESCRIPTION
**Pull request by JAdata**
Added (missing) test coverage for the fixup_pth_file() to catch the problem, then added a fix.

### CODE REVIEW

File: tests/test_fixup_scripts.py [hunk 1]
```diff
@@ -187,3 +190,28 @@ def test_fixup_script_different_version(self):
...
+
+        # Find size of file - should be plain ASCII/UTF-8...
+
+        orgStat = os.stat(pth)
+        assert len(content) == orgStat[stat.ST_SIZE]
+
+        # Now sleep for 2 seconds, then call fixup_pth_file(). This should ensure that the stat.ST_MTIME has
+        # changed if the file has been changed/rewritten
+
+        time.sleep(2)
```
Comment (timabbott) [inline tests/test_fixup_scripts.py]: Can you instead either set a timestamp in the past or mock the `os.stat` call to provide a later?  A 2-second sleep makes the tests slower, so I'd like to avoid adding it to the test suite..

File: clonevirtualenv.py [hunk 2]
```diff
@@ -10,7 +11,7 @@
 import sys
 import itertools
 
-version_info = (0, 2, 6)
+version_info = (0, 2, 7)
```
Comment (timabbott) [inline clonevirtualenv.py]: Bumping the version should be a separate commit that we do when releasing; can you remove this?

**Comment by timabbott** [comment]
Thanks for working on this @JAdata!  I posted a few comments.

File: tox.ini [hunk 3]
```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27,py26,py25,py32
+envlist = py27,py33,py34,py35,py36
```
Comment (timabbott) [inline tox.ini]: This change is great, but can you make it a separate commit?

File: tests/test_fixup_scripts.py (hunk 1 above)
Comment (JAdata) [reply tests/test_fixup_scripts.py]: The test suite already runs for 58 seconds. I would argue that those 2 extra seconds is well worth it for simplicity rather than adding of some obscure fiddling with timestamps or mocking. KISS :-)

File: clonevirtualenv.py (hunk 2 above)
Comment (JAdata) [reply clonevirtualenv.py]: Can do, but this is the wrong way of doing it - IMHO. The very first change I normally do to a module/package after deploying, is to bump the version number. The second after you have deployed you should never refer to the any new changes as a continuation of the old version. Changing the version should be the FIRST change you do, not the last...

File: tox.ini (hunk 3 above)
Comment (JAdata) [reply tox.ini]: Will redo the commits tomorrow from work...

File: clonevirtualenv.py (hunk 2 above)
Comment (timabbott) [reply clonevirtualenv.py]: Yeah, I don't disagree (ideally, actually, it'd be something between 0.2.6 and 0.2.7 for the Git versions).  But I only just got involved in helping maintain this the last couple weeks, and regardless, it should be its own commit.

**Comment by JAdata** [comment]
To achieve the requested split up of certain commits and avoid "commit havoc" in the history I have redone the commits in a new fork/branch and will issue a new pull request. Thus, closing this one.
